### PR TITLE
feat(deployment): add datetime output for ob-odc-web.std.log

### LINKED ISSUE #1353: [Enhancement]: add datetime output for ob-odc-web.std.log
### What would you like to be added?

 add datetime output for ob-odc-web.std.log.

### Why is this needed?

if no datetime output in log file, it is verfy hard to trouble shooting.

### PR DESCRIPTION
**Pull request by yizhouxw**
#### What type of PR is this?

type-feature

#### What this PR does / why we need it:
sometimes odc-server may crash, ob-odc-web.std.log offer valuable information for diagnose.
however there is no datetime info in ob-odc-web.std.log. this pr add datetime.

#### Which issue(s) this PR fixes:

Fixes #1353 

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: script/start-odc.sh [hunk 1]
```diff
@@ -54,17 +54,25 @@ function usage() {
     echo "- SERVER_PORT: deprecated environment variable, same as ODC_SERVER_PORT, for compatibility"
 }
 
+function log_info() {
+    echo 1>&2 "$(date +"%Y-%m-%dT%H:%M:%S.%Z") [INFO]" "$*"
```
Comment (smallsheeeep) [inline script/start-odc.sh:58]: Why do INFO level logs need to be redirected to standard error output as well?

Comment (yizhouxw) [reply script/start-odc.sh]: good question,  the code copy from where, here we may not redirect stdout to stderr.
fixed now.

File: server/odc-common/src/test/java/com/oceanbase/odc/common/security/SensitiveDataUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import java.util.Collection;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+
+@RunWith(Parameterized.class)
```
Comment (smallsheeeep) [inline server/odc-common/src/test/java/com/oceanbase/odc/common/security/SensitiveDataUtilsTest.java:28]: 👍🏻

**Review by smallsheeeep — APPROVED** [review]
LGTM

**Pull request merged**